3rdparty/CMakeLists.txt

```diff
@@ -99,6 +99,12 @@ else ()
   set(LIBRARY_SUFFIX ${CMAKE_STATIC_LIBRARY_SUFFIX})
 endif ()
 
+if (WIN32 AND BUILD_SHARED_LIBS)
+  set(3RDPARTY_LIBS_INSTALL_DIR ${MESOS_INSTALL_RUNTIME})
+else()
+  set(3RDPARTY_LIBS_INSTALL_DIR ${MESOS_INSTALL_LIBRARIES})
+endif()
+
```

Comment:
Earlier we agreed with @asekretenko that if he will not be on time by Tuesday this week to integrate his changes into master I will continue from the point where he left it (see above merge from his private fork on github). That was confirmed again in the morning before I started.

Once his PR is merged into master, you will see only my changes (in this PR).

There is no way for me to create a PR to Andrei's private branch (github doesn't give such option).

Also think about difficulty integrating those changes to DCOS packaging... there is another PR there that points to this branch with changes from both of us.

Reply:
@akornatskyy sure, I understand that you were going to continue work on top of Andrei's changes. I would propose that we do not review the changes in the current PR until Andrei's changes have landed, since it's difficult to review when the diffs are combined. Alternatively, you could open a PR just for review purposes against a branch in your private fork which include's Andrei's changes. I'm fine either way.

Reply:
sounds reasonable. let's wait for @asekretenko PR to land first.

Reply:
It looks like @asekretenko already incorporated the 3 commits from this PR that you mentioned, except for the addition of the `.jar` files to the installation. Andrei, is there a reason not to include the JARs?

Reply:
Oh good point, I can just review the changes in the last two commits - I'll do that, thanks!

Reply:
hm... just checked *master* branch of *apache/mesos* and *cmake_install_step_2* of *asekretenko/mesos* fork... the 3 commits are not there...

Reply:
@akornatskyy check out the patch set that I linked to on ReviewBoard: https://reviews.apache.org/r/72492/